src/modules/package-downloader.ts

```diff
@@ -40,7 +41,9 @@ export class PackageDownloader {
     }
 
     async checkoutVersion(): Promise<void> {
-        await this.git.checkout(this.packageConfig.version);
+        await this.git.checkout(
+            this.packageConfig.version.startsWith(BRANCH_PREFIX) ? this.packageConfig.version.substring(1) : this.packageConfig.version,
```

Comment:
Branch prefix can be longer than 1 character - you need to adjust substring to start after the length of BRANCH_PREFIX